recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+    - boost
+    - zlib
```

Comment:
Add `boost` and `zlib` to run requirements. You need to pin `boost` and `zlib`, see https://github.com/conda-forge/piranha-feedstock/blob/master/recipe/meta.yaml#L22-L24